js/views/startScreen.js

```diff
@@ -89,7 +89,7 @@ var StartScreen = React.createClass({
 
     var titleMetadata = (<div className={titleClass} style={titleStyle}>{this.props.contentTree.title}</div>);
     var iconName = (this.props.controller.state.playerState == CONSTANTS.STATE.END ? "replay" : "play");
-    var descriptionMetadata = (<div className={descriptionClass} ref="description" style={descriptionStyle}>{this.state.descriptionText}</div>);
+    var descriptionMetadata = (<div className={descriptionClass} ref="description" style={descriptionStyle}>{this.props.contentTree.description}</div>);
```

Comment:
this.props.contentTree.description has a [defaultProp](https://github.com/ooyala/html5-skin/blob/stable/js/views/startScreen.js#L169). 

@pilievOoyala with React.js defaultProps add stability to components and prevents us from having to add a bunch of null checks everywhere.

Also explained here - https://github.com/ooyala/html5-skin/pull/568#r67448769